src/Main.java

```diff
@@ -4,15 +4,12 @@ public static void main(String[] args) {
         Cart firstShoppersCart = new Cart();
 
 
-        Product sneakers = new Product(10);
-
-        try {
-            firstShoppersCart.addProduct( sneakers );
-        } catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
-        System.out.println( firstShoppersCart.getTotal() );
-
+//        try {
```

Comment:
Why we added this? Maybe as part of this method, we should just implement a very simple cart creation with a few products and then print it to the console?